frontend/src/components/VSkeleton/VGridSkeleton.vue

```diff
@@ -44,19 +44,22 @@ export default defineComponent({
     },
     numElems: {
       type: Number,
-      default: function () {
-        if (this.isForTab === "all") return 20
-        if (this.isForTab === "image") return 30
-        return 8
-      },
     },
   },
-  setup() {
+  setup(props) {
     function getRandomSize(max = 300, min = 100) {
       return Math.floor(Math.random() * (max - min) + min)
     }
 
-    return { getRandomSize }
+    // Calculate the default element count based on isForTab
+    const elementCount = computed(() => {
+          if (props.numElems) return props.numElems;
+          if (props.isForTab === "all") return 20;
+          if (props.isForTab === "image") return 30;
+          return 8;
+        });
```

Comment:
```suggestion
      if (props.numElems) return props.numElems
      if (props.isForTab === "all") return 20
      if (props.isForTab === "image") return 30
      return 8
    })
```

This is the change that the linter makes. The CI should pass if you accept it, @Ashhar-24 .
Did you run `just install` or `just node-install` before running the `just lint` command? It seems that the reason you are getting the error locally is the fact that not all dependencies are installed.